ioSPI/fourier.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
+import pyfftw.interfaces.numpy_fft
+from numba import jit
+
+
+@jit
+def make_neg_pos_2d(arr2d_shape):
```

Comment:
Why not `make_checkerboard`?

Reply:
I think this is a better name, too. Can we make that change?

Reply:
ok